docs/source/samplers.rst

```diff
@@ -86,3 +86,22 @@ High-Volubility sampler
 .. clidoc::
 
    child-project sampler /path/to/dataset /path/to/destination high-volubility --help
+
+Conversation sampler
+~~~~~~~~~~~~~~~~~~~~
+
+The conversation sampler returns the conversational blocks with the highest amount of turns.
+The first step is the detection of conversational blocks.
+Two consecutive vocalizations are considered part of the same conversational block if they are not separated
+by an interval longer than a certain duration, which by default is set to 1000 milliseconds.
```

Comment:
should we perhaps clarify that this is only for the ACLEW pipeline? if I read the code correctly above, the filter for 1k ms is not applied to subset from the TIFR/TIMR

Reply:
done!